app/models/constants/case_log.rb

```diff
@@ -77,23 +85,23 @@ module Constants::CaseLog
 
   NATIONAL = {
     "UK national resident in UK" => 1,
-    "A current or former reserve in the UK Armed Forces (excluding National Service)" => 100,
+    # "A current or former reserve in the UK Armed Forces (excluding National Service)" => 100,
```

Comment:
Is this meant to be commented out?

Reply:
we don't have it as an option any more. now removed

Reply:
Just to confirm is that change intended to be for this years form or next years form? (@paulrobertlloyd ) because changing that this year surely changes data continuity?

Reply:
As far as I can tell, this option shouldn’t be in this list, for either collection year.

Reply:
These are the nationality options from the 2021/22 form:

<img width="469" alt="Screenshot 2022-02-21 at 11 01 19" src="https://user-images.githubusercontent.com/813383/154942571-c75c2a69-99b6-4899-9481-3cb23177ed64.png">

Reply:
:man_facepalming:  Totally missed that this was the nationality question :+1: